package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "node build/dev-server.js",
     "build": "node build/build.js",
     "prod": "npm run build && npm start",
-    "start": "./node_modules/.bin/http-server -p ${BACKOFFICE_PORT:=3000} dist/",
+    "start": "./node_modules/.bin/start --port ${BACKOFFICE_PORT:=3000}",
```

Comment:
Because you added `rc`, for overriding the port, I think the environment variable is something like `KUZBO_PORT` https://www.npmjs.com/package/rc#standards